community/cython/APKBUILD

```diff
@@ -45,11 +45,20 @@ package() {
 _py2() {
 	cd "$builddir"-py2
 	_py python2
+
+	for f in cygdb cython cythonize; do
+	  mv "$subpkgdir"/usr/bin/$f "$subpkgdir"/usr/bin/${f}2
+	done
 }
 
 _py3() {
 	cd "$builddir"
 	_py python3
+
```

Comment:
The same `local f` improvement as above can be used here.

Reply:
Implemented.